app/views/dashboard/_user_profile.html.erb

```diff
@@ -3,47 +3,9 @@
     <%= gravatar(user) %>
     <%= link_to 'Edit Account', [:edit, @user], class: 'dashboard_profile_user-details_edit__btn' if policy(@user).edit? %>
   </div>
-  <div class="dashboard_profile_trades">
-    <div class="dashboard_profile_trades__table">
-      <div class="dashboard_profile_trades__row-headings">
-        <div class="dashboard_profile_trades__cell">Trades</div>
-        <div class="dashboard_profile_trades__cell">Received</div>
-        <div class="dashboard_profile_trades__cell">Allowed</div>
-      </div>
-      <% @user.trades_received_and_allowed_by_rarity.each do |trade| %>
-        <div class="dashboard_profile_trades__row">
-         <div class="dashboard_profile_trades__cell"><%= trade[:rarity].capitalize %></div>
-         <div class="dashboard_profile_trades__cell">
-          <input type="number" value="<%= trade[:num_received] %>" name="rarity-<%= trade[:rarity] %>" label="<%= trade[:rarity].capitalize %>" data-rarity="<%= trade[:rarity] %>" min="0" max="<%= trade[:num_allowed] %>"/>
-         </div>
-         <div class="dashboard_profile_trades__cell"><%= trade[:num_allowed] %></div>
-        </div>
-      <% end %>
-    </div>
-  </div>
+  <%= react_component("TradeTracker", {
+    user_id: @user.id,
+    current_user_id: current_user.id,
+    trades_data: @user.trades_received_and_allowed_by_rarity
+  }, {:style => "width: 100%"}) %>
```

Comment:
Probably work to move the entire user page to react soon so we don't have random parts in react all over the place but for now this works for the transition period